package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-jablotron",
-  "version": "3.1.4",
+  "version": "3.1.5",
```

Let's bump at least the minor instead of patch:
```suggestion
  "version": "3.2.0",
```